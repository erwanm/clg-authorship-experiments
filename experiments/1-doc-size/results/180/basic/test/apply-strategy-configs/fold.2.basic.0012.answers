1903-us-tjoas 1854-hbs-smofl	0
1895-wta-alae 1906-mt-cfma	0.537734
1891-fmc-ttf 1881-haj-fcbtp	0
1872-haj-ptf 1906-rwc-ttolp	0.610871
1898-hj-ttm 1879-hj-h	0.381972
1874-wta-tcw 1900-haj-adoh	0.404212
1878-hj-te 1882-lma-ajsb-v6	0.10714
1883-hj-rh 1898-hj-itc	0.410472
1885-hj-gr 1910-us-sts	0.731569
1914-us-sm 1897-ewaoc-tdof	0.04821
1896-espw-cfal 1887-espw-jtf	0.424821
1887-espw-tgb 1900-espw-wtg	0.698089
1894-cdw-tgh 1907-fmc-tdr	0.905874
1900-fmc-itpotk 1905-fmc-fm	0.233784
1889-fmc-g 1898-haj-tyb	0.280723
1890-hj-ttm 1904-hj-tgb	0.195612
1878-hbs-pp 1870-hbs-tbc	0.885313
1901-us-km 1921-us-tbol	0.557529
1883-hj-rh-v2 1872-wta-lb	0.073589
1866-haj-ccc-ia 1871-hbs-mwai	1
1880-hj-c 1885-hj-altof	0.708611
1898-fmc-ari-v2 1893-fmc-kl-v2	0.750379
1918-rwc-tlg 1863-wta-isao	0.677911
1884-ab-sbts 1898-us-apom	0.291112
1889-cfw-jl 1915-rwc-p	0.528154
1872-mt-ri 1895-haj-tdw	0.708008
1902-fmc-c 1883-fmc-tl	0.298001
1859-hbs-tmw 1887-fmc-pp	0.83611
1884-wta-Sac 1882-wta-aa	0.578535
1871-haj-tt 1876-wta-vao	1
1913-es-tm 1911-rwc-tgr	0.73352
1905-es-tpoe 1907-es-tlotg	1
1899-haj-mmv 1866-haj-hf	0.936536
1891-fmc-do 1876-cdw-itl	0.4035
1892-fmc-pg 1883-fmc-dc	0.792491
1917-ab-bn 1922-us-tcmc	1
1889-cdw-sths 1869-espw-mwag	0.84218
1902-hj-twotd 1896-hj-tsop	0.642048
1888-haj-bb 1899-haj-ra	1
1913-ew-tcotc 1868-haj-rd	0.659418
1868-espw-tga 1919-rwc-tsos	1
1900-haj-ainy 1919-ew-m	0.043335
1881-mt-tpatp 1899-haj-aoatb	0.400731
1891-fmc-k 1893-fmc-tr	0.506694
1881-hj-tpoal 1890-fmc-acmr	0.801561
1869-haj-mtmb 1903-haj-bba-ph	1
1885-fmc-atoalp 1905-fmc-wso	0.724403
1868-wta-mob 1870-wta-faf	0.47745
1871-hbs-pawt 1904-rwc-isotu	0.318438
1875-cfw-cn 1864-lma-m	1
1914-mh+-mhce-nf 1908-fmc-tp	0.592394
1893-fmc-md 1895-fmc-ajs	0.560096
1864-wta-tsb 1883-wta-sh	0.790843
1900-rwc-taoamm 1910-rwc-tgm	1
1903-fmc-thor 1887-fmc-s	0.632189
1884-mt-taohf 1874-mt-lotm	0
1907-espw-tldup 1871-espw-tsp	0.858981
1866-espw-gcj 1882-haj-bn	1
1883-haj-tbb 1888-es-ttatv	0.757146
1908-fmc-s 1908-fmc-tws	0.815086
1889-wta-wtel 1911-rwc-tcl	0.30517
1904-rwc-sm 1886-hj-pc	0.608962
1918-us-jh 1897-ew-tdof	0.418965
1872-lma-ss 1867-lma-tagomtt	0.52804
1910-ew-tomag 1889-es-atg	0.930888
1889-espw-tsfi 1895-espw-tsojc	0.440153
1905-haj-fftf 1870-haj-rar	1
1882-espw-dz 1909-mt-isd	0.954485
1907-ew-tfott 1896-hj-toh	0.256317
1868-wta-dad 1885-wta-sts	1
1909-rwc-tdm 1897-ab-tdohy	0.662093
1891-haj-tetb 1868-haj-faf	0.892332
1893-haj-dtn 1866-haj-tcw	1
1869-wta-dtr 1865-wta-waw	0.433577
1891-cdw-oi 1873-hbs-wish	0.687495
1879-hj-dm 1903-hj-ta	1
1874-cfw-tac 1882-fmc-mi	1
1893-espw-dm 1896-ab-boat	0.754847
1896-espw-tsasa 1894-espw-asl	0.983726
1879-haj-tym 1869-haj-pjb	1
1920-us-1tsoap 1913-us-dg	0.656601
1917-es-ow-sh 1873-hbs-pl	0.761777
1855-wta-aa 1873-wta-tyc	0.871045
1879-haj-ttb 1894-fmc-wg	1
1898-fmc-vc 1895-fmc-t	0.868139
1867-espw-gyatgc 1901-espw-a	1
1896-fmc-c 1885-fmc-zamc	0.911836
1855-hbs-tmf 1866-espw-gb	0.419895
1897-fmc-aroy 1884-fmc-aap	0.458753
1883-espw-btg 1911-hj-to	0.95289
1862-hbs-tpooi 1875-hbs-waon	1
1883-fmc-ars 1885-fmc-tcotk	0.096813
1886-hbs-tsw 1854-hbs-smofl-v2	0.836536
1903-ew-ivatg 1878-cdw-hikab	0.104159
1858-wta-pap 1897-haj-fof	0.256989
1902-hj-twotd-v2 1888-hj-tr	0.221052
1871-hbs-ofs 1888-fmc-si	0.377789
1907-ab-rm 1901-hj-tsf	1
1882-haj-ffbts 1893-haj-ianwatgfoa	0.72159
1908-rwc-tfl 1910-rwc-ap	0.688542
