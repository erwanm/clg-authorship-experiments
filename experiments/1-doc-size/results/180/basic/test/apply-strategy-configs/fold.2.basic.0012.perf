0.325584	0.581400	0.560000
