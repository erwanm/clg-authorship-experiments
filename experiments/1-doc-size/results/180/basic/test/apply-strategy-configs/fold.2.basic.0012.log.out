apply-config.sh: applying config '/tmp/expe1/180/process//selected-meta-configs/fold.2.strategy-as-meta.basic.0012.model/strategy-configs/fold.2.basic.0012' to cases '/home/moreaue/dump/expe1//results/180/basic/test/apply-strategy-configs/only-cases.tmp.bhzzkSOb'; results written to '/home/moreaue/dump/expe1//results/180/basic/test/apply-strategy-configs/fold.2.basic.0012'
apply-config.sh: done.
